packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -62,6 +63,22 @@ export class Lb3AppBooter implements Booter {
       this.mountRoutesOnly(lb3App, spec);
     }
 
+    const dataSources = lb3App.dataSources;
+    if (dataSources) {
+      const processedDs: string[] = [];
+      Object.keys(dataSources).forEach(key => {
+        key = key.toLowerCase();
+        if (!processedDs.includes(key)) {
```

Comment:
Let's use reverse logic please.

```ts
if (processedDs.includes(key)) continue;
```

Is it safe to use `key.toLowerCase()` to decide whether two entries are unique? I think it would be much safer to compare the actual datasource instances (by reference).

```ts
Object.values(dataSources).forEach(ds => {
  if (visited.includes(ds)) return;
  visited.push(ds);

  // ...
});
```